todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/statistics/StatisticsActivity.java

```diff
@@ -38,7 +38,7 @@
 public class StatisticsActivity extends DaggerAppCompatActivity {
 
     @Inject
-    StatisticsPresenter mStatiticsPresenter;
+    StatisticsPresenter mPresenter;
     @Inject
     StatisticsFragment fragment;
```

Comment:
I think that this PR is missing something... I think you you could also change names of fragments in `Activity` classes to name with hungarian notation like `mFragment`. I saw that in `TaskDetailActivity` there is also 'injectedFragment', a bit strange name for `TaskDetailFragment` :-)

This change would make this PR more consistent because it's related to `binding functions` in the name of request.

@digitalbuddha what do you think about it?